tests/devfs/main.c

```diff
@@ -23,9 +23,7 @@
 #include "fs/devfs.h"
 #include "random.h"
 
-#include "embUnit/embUnit.h"
-
-#include "tests-devfs.h"
```

Comment:
Since you removed the include, I guess the header you moved can also be removed instead.

Reply:
Oops, I thought I removed it.